### DISCUSSION TITLE
Manually select HERO art?

### DISCUSSION THREAD

**Discussion by billyors** [lostb1t/replex#64] [Q&A] [answered]
🤣 Is there a way to modify the HERO artwork manually? The Flash (2023) is appearing as Flesh . Can confirm it is matched properly and the metadata appears to work fine from the default view. HERO art seems to be the only issue.

![Screenshot 2023-08-28 161156](https://github.com/sarendsen/replex/assets/69538148/0f6c7529-f97f-4ae2-8511-af72db38a5a4)

**Comment by jl94x4** [comment, marked as answer]
#62
